NumericInput/NumericInput.js

```diff
@@ -32,6 +32,10 @@ export default class NumericInput extends Component {
     updateBaseResolution = (width, height) => {
         calcSize = create({ width, height })
     }
+    
+    startInc = () => {this.inc(); this.incInterval = setInterval(this.inc, 100); }
+    startDec = () => {this.dec(); this.decInterval = setInterval(this.dec, 100); }
```

Comment:
please change them to be spread on a few lines, like the other functions, please

Reply:
since they have more than one command
also please remove the semicolon from the end of lines, please